src/agent/windows/WMICommunication.h

```diff
@@ -4,26 +4,39 @@
 #include <memory>
 #include <Wbemidl.h>
 #include "common/SmartData.h"
+#include "../Disk.h"
 
 
 
 class WMICommunication
```

Comment:
Just to consider: one base class for common stuff which can be inherited and used by specialized classes

Reply:
Issue #39 added. Will be addressed in future.